src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py

```diff
@@ -8,6 +8,7 @@
 from msrestazure.azure_exceptions import CloudError
 from msrestazure.tools import resource_id, is_valid_resource_id, parse_resource_id  # pylint: disable=import-error
 from knack.log import get_logger
+import psycopg2
```

Comment:
could you check if there is packaging issue for darwin like here?
https://github.com/Azure/azure-cli-extensions/blob/master/src/db-up/azext_db_up/custom.py#L193-L216

Reply:
Package tested there is: pymssql.  This one is psycopg2 and db-up (from what I saw) doesn't have a darwin issue with this package.  CI + locally didn't notice any issue with this either aside from having to add postgresql-dev to docker.